draft-ietf-httpbis-resumable-upload.md

```diff
@@ -383,19 +378,6 @@ upload-draft-interop-version: 3
 :status: 204
 ~~~
 
```

Comment:
The deleted block is consistent with the normative changes above and so might need some discussion.

Reply:
I'm OK with the removal since it's also a remanent of the original Upload-Token approach, but @Acconut thought we should keep it.

Reply:
I still think this is a value section for developers implementing this draft. However, maybe such advice is better left in some other documents, so I am fine with the removal as well.